### ISSUE TITLE
`chown`ing not exists $symlink

### ISSUE THREAD

**Issue by robi50** [andres-montanez/Magallanes#199]
Release task fails if "current release" symbolic link directory not exists since it will try to `chown` not exists directory. Command fails with "No such file or directory"

```yaml
deployment:
  user: robin
  from: ./
  to: /home/robin/mage-test
releases:
  enabled: true
  max: 10
  symlink: current
  directory: releases
```
2015-03-02 23:21:13 -- ---- Executing: $ ssh  -p 22 -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no robin@{IP_ADDRESS} "sh -c \"cd /home/robin/mage-test && chown -h robin:robin current && chown -R robin:robin releases/20150302232106 && chown robin:robin releases\""
2015-03-02 23:21:14 -- chown: cannot access 'current': No such file or directory
2015-03-02 23:21:14 -- ---------------------------------
2015-03-02 23:21:14 -- FAIL
2015-03-02 23:21:14 -- Finished the Releasing
2015-03-02 23:21:14 -- No Post-Deployment tasks defined.
2015-03-02 23:21:14 -- Time for deployment: 5 seconds.
2015-03-02 23:21:14 -- Average time per host: 5 seconds.
2015-03-02 23:21:14 -- Total time: 8 seconds.
2015-03-02 23:21:14 -- Finished Magallanes

**Comment by spongeben** [comment]
There's a PR about this bug #195 (first part), but it needs feedback if you can have a look ;)

**Comment by eps90** [comment]
I'm closing this issue (actually fixed in #195).
